docusaurus/docs/overview/feature-model/index.mdx

```diff
@@ -47,20 +41,42 @@ Overture data is described by the [_simple feature model_](https://www.ogc.org/s
 
-## Common definitions and specifications
-Some feature properties and attributes have common definitions and specifications that can be quite complex.
+The Overture corpus has a core set of properties for every feature.
+- `theme`: 
+themes are top-level categories of map features.
+- `type`: a theme may have multiple types of features; the transportation theme, for example, has both segments and connectors.
+- `sources`: this is the array of information about the data sources from which a feature's attributes originated.
+- `version`: the version number of a feature is incremented in each Overture release where the geometry or attributes of the feature change.
+- `updateTime`: the timestamp indicates when the feature was last updated.
+
+Different theme and type values will contribute additional properties to the schema. For example, in the building described above, the buildings `theme` with building `type` has a `class` property that defines what kind of building it is, in this case a civic building. It also has an optional `height` property. You can find more detail about the properties associated with each theme and type in the Schema Themes and Schema Reference sections of this documentation.
+
+
+## Complex attributes
+
+Some feature properties are quite complex and require further explanation in the documentation.
+
+- **[Global Entity Reference System (GERS) IDs](../../gers/gers.mdx)**: the [Global Entity Reference System](/gers) is responsible for handling persistent identification across entities in the Overture corpus.
+- **[Names](names)**: where applicable, complex attributes such as names will always use the same schema across all themes.
+- **[Scoped and rule-based properties](scoping-rules)**: a scoping property narrows the scope of its parent property and may only apply to a particular instance of the parent property; rule-based property is an array of rules, which are objects containing at lease one scoping property.
+
+
+## Extensible attributes
+
+Any additional top-level property prefixed with `ext` may be added to an Overture feature, usually via the conflation of an external dataset and an Overture dataset.
+
+## Measurements
 
-- [Global Entity Reference System (GERS) IDs](../../gers/gers.mdx)
-- [Names](names)
-- [Scoped and rule-based properties](scoping-rules)
+Measurements in the schema follow [The International System of Units (SI)](https://www.bipm.org/en/publications/si-brochure), e.g. meters for building heights. Regulations and restrictions encoded in the schema follow local units. Speed limits, for example, appear as miles per hour (mph) in features located in the United States. Features in Germany show kilometers per hour (kph) for speed limits.
```

Comment:
This I would characterize a little bit differently.

I would say that:

1. Objective measurements of things in the world follow SI, meaning heights, widths, lengths, *etc.* that represent real-world properties *of features*.
    - In the schema, these values are provided as scalar numeric value without units, since the units are implied and documented in place in the schema.
    - The reason we do this is to maximize consistency and predictability.
3. Quantities specified in regulatory rules, norms, and customs follow local specifications wherever possible.
    - In the schema, these values are provided as two-element arrays where the first element is the scalar numeric value and the second value is the units, *e.g.* `[100, "km/h"]` <--- double check this is correct though.
    - The reason we use the local units is to ensure that norms are specified in the most relevant way to the people who are actually there.
    - The reason we include the units is to maximize consistency and predictability in the schema.

Those are just the rough cliff notes around how things are done and why.